pkg/operator/ceph/cluster/mgr/orchestrator.go

```diff
@@ -55,9 +55,10 @@ func (c *Cluster) setRookOrchestratorBackend() error {
 		orchestratorCLIName = orchestratorNewCLIName
 	}
 	// retry a few times in the case that the mgr module is not ready to accept commands
-	_, err := client.ExecuteCephCommandWithRetry(func() ([]byte, error) {
+	_, err := client.ExecuteCephCommandWithRetry(func() (string, []byte, error) {
 		args := []string{orchestratorCLIName, "set", "backend", "rook"}
-		return client.NewCephCommand(c.context, c.Namespace, args).RunWithTimeout(client.CmdExecuteTimeout)
+		output, err := client.NewCephCommand(c.context, c.Namespace, args).RunWithTimeout(client.CmdExecuteTimeout)
+		return "set rook backend", output, err
```

Comment:
ditto

Reply:
What about `errors.Wrapf(err, "set rook backend. %s", output)`?

Reply:
The output isn't part of the error. The caller may need to analyze the output when it's successful